lib/model/model.go

```diff
@@ -2096,21 +2096,38 @@ func stringSliceWithout(ss []string, s string) []string {
 // list, we don't also need foo/bar/ because that's already covered.
 func unifySubs(dirs []string, exists func(dir string) bool) []string {
 	var subs []string
+	sep := string(filepath.Separator)
```

Comment:
Not able to figure out what you are referring to. 
Are you referring to line 2107? 
If yes then its a deleted line..
If not can you be more specific...